src/todo/todointerfaces.ts

```diff
@@ -32,6 +38,11 @@ export interface ITodoDetails {
   dueAt?: Date;
   subInstanceId?: string;
   token?: string;
+  // Additional data a todo can contain, may depend on the module
+  data: {
+    // The risk assessment cycle of the risk at the time the todo was created
+    riskAssessmentCycle?: RiskAssessmentCycle;
```

Comment:
Maybe it is better in the future if we extend `ITodoDetails` and create `ITodoDetailsRiskAssessment` that then contains this field. I don't know if this would make code clearer in usage.